cli/command/registry/login.go

```diff
@@ -41,7 +41,7 @@ func NewLoginCommand(dockerCli command.Cli) *cobra.Command {
 	flags := cmd.Flags()
 
 	flags.StringVarP(&opts.user, "username", "u", "", "Username")
-	flags.StringVarP(&opts.password, "password", "p", "", "Password")
+	flags.StringVarP(&opts.password, "password", "p", "", "Password (deprecated, use --password-stdin)")
```

Comment:
Hm, is it officially deprecated? If so, it needs to go through the deprecation steps; https://github.com/docker/cli/blob/master/docs/deprecated.md

Given that this flag has been around for a _long_ time, I'm not sure we can deprecate it (at least, it would require a really long deprecation period)

FWI, the `pflags` package allows marking a flag as deprecated, which will hide it, and print a message if used